src/System.Private.Uri/src/System/UriFormatException.cs

```diff
@@ -2,12 +2,22 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if netstandard10
```

Comment:
It really isn't netstandard1.0 complain as those APIs aren't present there they only exist in the latest version. We will soon be moving these type of libraries to netcoreapp specific targets (which is what this guy should probably be packages as). 

cc @ericstj